subsys/bluetooth/audio/bap_iso.h

```diff
@@ -16,7 +16,7 @@ struct bt_bap_iso_dir {
 	struct bt_bap_ep *ep;
 	struct bt_iso_chan_path path;
 	struct bt_iso_chan_io_qos qos;
-	uint8_t cc[CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_COUNT * CONFIG_BT_AUDIO_CODEC_MAX_DATA_LEN];
+	uint8_t cc[CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE];
```

Comment:
Since this is an internal header, I suggest the former. The reason why we guard them in the public header, is that the Kconfig values may not exist at all, in which case we cannot simply use them for the array size without a guard, but we can do that here